ui/app/build-configs/detail/details-tab/pncBuildConfigDetailsEditor.js

```diff
@@ -24,6 +24,10 @@
        * Object: The buildConfig object to be edited
        */
       buildConfig: '<',
+      /**
+       * Object: The detailed productVersion object of the build config
+       */
+      productVersion: '=',
```

Comment:
Hi Martin, the reason I am using the = is for updating the productVersion object after editing the BC, otherwise the changes to the BC will not be shown until user refresh the page manually.

Reply:
So at this point I think there are two possible solutions:
 - automatically reload the page after changes are submitted, we are using the same approach on several other pages (like https://github.com/project-ncl/pnc/blob/master/ui/app/product-milestones/components/pnc-product-milestone-actions/pncProductMilestoneActions.js#L68 )
 
 - using `&` binding you could update state in parent scope using method

Reply:
@matedo1 
Hi Martin, 
After I tried both solutions I think the first one would be better. Just updated the code and feel free to have a look at any time. 

Thanks!
Dennis